hadoop-ozone/dist/pom.xml

```diff
@@ -105,6 +105,16 @@
                   <artifactId>protobuf-java</artifactId>
                   <version>${grpc.protobuf-compile.version}</version>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.hadoop</groupId>
+                  <artifactId>hadoop-auth</artifactId>
+                  <version>${hadoop2.version}</version>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.hadoop</groupId>
+                  <artifactId>hadoop-annotations</artifactId>
+                  <version>${hadoop2.version}</version>
+                </artifactItem>
```

Comment:
We should not copy Hadoop2 jars to `share/ozone/lib`.  These are expected to be provided by users who wish to use Ozone in a Hadoop2 environment.